src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -237,6 +250,32 @@ def _reset(self):
         self.log.debug("Remote scheduler option:", self.remoteScheddOptions)
         return S_OK()
 
+    def _executeCondorCommand(self, cmd, tokenFile=None, keepTokenFile=False):
```

Comment:
I am not sure to understand this method.
For instance, if we want to get the status of 50 pilots: `getJobStatus()`:
- we have `tokenFile = None`
- because it is `None`, it will create 50 token files which are not returned by the method
- None of them will be deleted because `tokenFile` is still `None` at the end.

Am I missing something here?

I would have created a `_prepareTokenFile(): str (tokenFile)` method that I would have used only `self.token` exists and I would have deleted it manually.

Reply:
Yes, it looks to be a flaw in the logic. I have updated this part of the code.